osu.Game.Tests/Resources/TestResources.cs

```diff
@@ -85,7 +85,8 @@ public static string GetTestBeatmapForImport(bool virtualTrack = false)
         /// </summary>
         /// <param name="difficultyCount">Number of difficulties. If null, a random number between 1 and 20 will be used.</param>
         /// <param name="rulesets">Rulesets to cycle through when creating difficulties. If <c>null</c>, osu! ruleset will be used.</param>
-        public static BeatmapSetInfo CreateTestBeatmapSetInfo(int? difficultyCount = null, RulesetInfo[] rulesets = null)
+        /// <param name="randomiseMetadata">Whether to randomise metadata to create a better distribution.</param>
+        public static BeatmapSetInfo CreateTestBeatmapSetInfo(int? difficultyCount = null, RulesetInfo[] rulesets = null, bool randomiseMetadata = false)
```

Comment:
Not too hot on the metadata randomisation reaching all the way out to here. Would prefer doing it locally to the only usage for now.